digdag-docs/src/operators/py.md

```diff
@@ -76,6 +76,111 @@ See [Python API documents](../python_api.html) for details including variable ma
     print("simple execution")
   ```
 
+  You can pass arguments to class for initialization　by defining arguments under the `py>:` operation as the following:
+  ```yaml
+  # sample.dig
+  +some_task:
+    py>: tasks.MyWorkflow.my_task
+    required1_1: awesome execution
+    required1_2: "awesome execution"
+    required2: {a: "a"}
+    required3: 1
+    required4: 1.0
+    required5: [a, 1, 1.0, "a"]
+  ```
+
+  Also, you can do the same thing using `_export` as the following:
+  ```yaml
+  # sample.dig
+  +some_task:
+    _export:
+      required1_1: awesome execution
+      required1_2: "awesome execution"
+      required2: {a: "a"}
+      required3: 1
+      required4: 1.0
+      required5: [a, 1, 1.0, "a"]
+    py>: tasks.MyWorkflow.my_task
+  ```
+
+  This example assume following Python script:
+
+  ```python
```

Comment:
This example does not work by copy and paste it.
Don't we need to  import `Union` as follows ?
```
from typing import Union

class MyWorkflow(object):
...
```